test/Microsoft.Framework.ConfigurationModel.FunctionalTests/project.json

```diff
@@ -2,6 +2,7 @@
   "dependencies": {
     "Microsoft.Framework.ConfigurationModel.Json": "1.0.0-*",
     "Microsoft.Framework.ConfigurationModel.Xml": "1.0.0-*",
+    "Microsoft.Framework.ConfigurationModel.Test.Common": "1.0.0-*",
```

Comment:
I overwrote the `Equals` method so that `X < T`. Is there a problem? :trollface: 

PS: I'll fix this in my next commit